src/components/Container.jsx

```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components';
 
 const Container = styled.div`
   width: 90%;
-  max-width: 1000px;
+  max-width: 906px;
```

Comment:
Container should remain 1000px
```suggestion
  max-width: 1000px;
```